src/SFA.DAS.EmployerAccounts.Web/Helpers/ControllerConstants.cs.cs

```diff
@@ -0,0 +1,78 @@
+﻿namespace SFA.DAS.EmployerAccounts.Web.Helpers
```

Comment:
Err this whole file looks wrong; needs any changes we want applying to the correct file and this file removing.

Reply:
Its not showing as .cs.cs in my project?

Reply:
It won't show in the project as it is not a valid code file, but it is on disc. Looks like some merge artefact as it has the `HEAD>>>` markers in it.

Reply:
Removed this file.